source/framework/sensitivity/src/TRestComponentFormula.cxx

```diff
@@ -231,6 +237,9 @@ void TRestComponentFormula::InitFromConfigFile() {
 
     if (!fFormulas.empty()) return;
 
+    /// For some reason I need to do this manually. Dont understand why!
+    fFormulaUnits = GetParameter("formulaUnits");
```

Comment:
@nkx111 may be you know why I needed to use this? This line seemed to solve my problems with `formulaUnits` initialization from config file. However, it seems to be working properly for all members inside `TRestAxionHelioscopeSignal` that also inherits from `TRestComponent`.